models/Sherlock-Benchmark-10-Unicycle/README.md

```diff
@@ -1,4 +1,5 @@
-# Unicycle Car Model
+# Sherlock Benchmark 10 Unicycle Car
```

Comment:
```suggestion
# Unicycle Car
```
(i suggest to keep the Sherlock .. name in the text)

"This model of a unicycle car is taken from Sherlock's tool Benchmark 10"